subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -374,7 +397,12 @@ private void parseData(String data) {
           } else {
             // running configuration requests
             if (data.indexOf(device_hostname) >= 0 && data.length() < 20) {
```

Comment:
use named constant (for 20)

Reply:
fixed, implemented constant shortPacketLength